dateutil/test/test_parser.py

```diff
@@ -152,7 +152,7 @@ def test_parser_default(parsable_text, expected_datetime, assertion_message):
     assert parse(parsable_text, default=datetime(2003, 9, 25)) == expected_datetime, assertion_message
 
 
-class TestFormat(unittest.TestCase):
+class TestFormat(object):
```

Comment:
I've been writing so much Python 3 lately that I was initially taken aback by this. Sooooooon.